### PR TITLE
no file copy prior to upload

### PR DESCRIPTION
**Pull request by curlup**

### CODE REVIEW

File: cli/upload.py [hunk 1]
```diff
@@ -83,65 +84,154 @@ def _open_file_mapping(extra_metadata: dict) -> Dict[str, BinaryIO]:
             f.close()
 
 
+_IGNORED_WARN_LINES = set(
```
Comment (jacoblurye) [inline cli/upload.py]: Where did this text come from? Can you add a comment?

File: cli/upload.py [hunk 2]
```diff
@@ -83,65 +84,154 @@ def _open_file_mapping(extra_metadata: dict) -> Dict[str, BinaryIO]:
...
+
+def _wait_for_upload(procs: list) -> bool:
+    """
+    Waits for all subprocesses and prints their stderr streams.
+    Returns bool - if an error has occurred during any if uploads
+    """
+
+    finished = set()
+    errored = False
+    while len(finished) != len(procs) and not errored:
```
Comment (jacoblurye) [inline cli/upload.py]: Is there a way to wait for subprocesses to finish without spinning in this way? If not, should this loop at least sleep briefly between iterations?

File: cli/upload.py [hunk 3]
```diff
@@ -83,65 +84,154 @@ def _open_file_mapping(extra_metadata: dict) -> Dict[str, BinaryIO]:
...
+
+    print(f"[{len(finished)}/{len(procs)} files]")
+    return errored
+
+
 def _gsutil_assay_upload(upload_info: api.UploadInfo, xlsx: str):
     """
     Upload local assay data to GCS using gsutil.
     """
-    workspace = _get_workspace_path()
```
Comment (jacoblurye) [inline cli/upload.py]: If the `workspace` directory is no longer used by the CLI, should we remove this concept from the CLI code entirely? E.g., here: https://github.com/CIMAC-CIDC/cidc-cli/blob/c2669fda3383e9c8978645ff6b7ad47a4bef0cb4/cli/config.py#L11

**Comment by jacoblurye** [comment]
Don't forget to bump the package version! Also, probably a good idea to update https://github.com/CIMAC-CIDC/cidc-api-gae/blob/451c0fb2fa54e3ed055cc61fee708abfd1f0caf8/cidc_api/config/settings.py#L19

File: cli/upload.py (hunk 2 above)
Comment (curlup) [reply cli/upload.py]: the idea is that `.stderr.readline()` (few lines down) will block until gsutil prints next "progress" line, which is couple of times per second, so this won't be a hot loop, I'd say.

I confirmed in a manual measurement that the loop waits in the `readline` call 99% of the time. 

What do you think?

File: cli/upload.py (hunk 3 above)
Comment (curlup) [reply cli/upload.py]: sure, we don't need it (for now at least - might be useful if we decide to work on resumable uploads though)

Comment (jacoblurye) [reply cli/upload.py]: Ok, no harm in leaving it in.

File: cli/upload.py (hunk 2 above)
Comment (jacoblurye) [reply cli/upload.py]: Sounds fine to me!

**Pull request merged**